FAQS.md

```diff
@@ -0,0 +1,54 @@
+**Markor - Frequently Asked Questions**
+
+**Markor - FAQ**
+
+This Frequently Asked Question will be continually updated to accommodate new information with respect to new updates and changes on the App and will be used inside the App to help users find answers to their questions.
```

Comment:
use "Project" instead of "App" wherever possible. E.g. "inside the app" is very ok, but e.g. "changes on the project"